parm/config/config.ecfvrfy

```diff
@@ -0,0 +1,198 @@
+        export VBACKUP_PRCP=24                          # back up for QPF verification data
+        export vsdbsave="$NOSCRUB/archive/vsdb_data"    # place to save vsdb database
+        export vsdbhome=$BASE_VERIF                     # location of global verification scripts
+        export VSDBSH="$vsdbhome/vsdbjob.sh"            # VSDB job script
+        export VSDBJOBSH=$SCRgfs/vsdbjob_submit.sh      # VSDB submission job
+        export vlength=$FHMAX_GFS                       # verification length
+        export vhr_rain=$FHMAX_GFS                      # verification length for precip
+        export ftyplist="pgbq"                          # verif. files used for computing QPF ETS scores
+        export ptyplist="PRATE"                         # precip types in GRIB: PRATE or APCP
+        export anltype="gfs"                            # default=gfs, analysis type (gfs or gdas) for verification
+        export rain_bucket=6                            # prate in pgb files is 6-hr accumulated
+
+        export VSDB_START_DATE="$SDATE"                 # starting date for vsdb maps
+        export webhost="emcrzdm.ncep.noaa.gov"          # webhost(rzdm) computer
+        export webhostid="$USER"                        # webhost(rzdm) user name
+        export SEND2WEB="NO"                            # whether or not to send maps to webhost
+        export WEBDIR="/home/people/emc/www/htdocs/gmb/${webhostid}/vsdb/$PSLOT"
+        export mdlist="gfs $PSLOT "                     # exps (up to 10) to compare in maps
+    fi
+fi
+
+
+#----------------------------------------------------------
+# Minimization, Radiance and Ozone Monitoring
+#----------------------------------------------------------
+
+if [ $VRFYRAD = "YES" -o $VRFYMINMON = "YES" -o $VRFYOZN = "YES" ]; then
+
+    # Radiance Monitoring
+    if [[ "$VRFYRAD" == "YES" && "$CDUMP" == "$CDFNL" ]] ; then
+
+        export RADMON_SUFFIX=$PSLOT
+        export TANKverf="$NOSCRUB/monitor/radmon"
+        export VRFYRADSH="$HOMEgfs/jobs/JGDAS_ATMOS_VERFRAD"
+
+    fi
+
+    # Minimization Monitoring
+    if [[ "$VRFYMINMON" = "YES" ]] ; then
+
+        export MINMON_SUFFIX=$PSLOT
+        export M_TANKverf="$NOSCRUB/monitor/minmon"
+        if [[ "$CDUMP" = "gdas" ]] ; then
+            export VRFYMINSH="$HOMEgfs/jobs/JGDAS_ATMOS_VMINMON"
+        elif [[ "$CDUMP" = "gfs" ]] ; then
+            export VRFYMINSH="$HOMEgfs/jobs/JGFS_ATMOS_VMINMON"
+        fi
+
+    fi
+
+    # Ozone Monitoring
+    if [[ "$VRFYOZN" == "YES" && "$CDUMP" == "$CDFNL" ]] ; then
+
+        export HOMEgfs_ozn="$HOMEgfs"
+        export OZNMON_SUFFIX=$PSLOT
+        export TANKverf_ozn="$NOSCRUB/monitor/oznmon"
+        export VRFYOZNSH="$HOMEgfs/jobs/JGDAS_ATMOS_VERFOZN"
+
+    fi
+
+fi
+
+
+#-------------------------------------------------
+# Cyclone genesis and cyclone track verification
+#-------------------------------------------------
+
+export ens_tracker_ver=v1.1.15.5
```

Comment:
Can we standardize the installations for the TC tracker so we don't need this confusing definition @KateFriedman-NOAA ? Different versions, different paths; it's a mess.

Reply:
My test version of config.vrfy for this branch on WCOSS2 consolidated the `HOMEens_tracker` (and `ens_tracker_ver`) settings to this:

```
export HOMEens_tracker=$BASE_GIT/TC_tracker/${tracker_ver}
```

I have been installing the newer copy of `TC_tracker` on each supported platform in a folder called `TC_tracker` with a subfolder (with the install) based on the version (`tracker_ver`). This matches how NCO installs packages on WCOSS2 (application/version). I also added `export tracker_ver=v1.1.15.5` to my `$target.ver` files.

See this on Cactus: `/lfs/h2/emc/global/save/emc.global/git/TC_tracker/v1.1.15.5`

My testing of this piece was put on hold to attend to WCOSS2 ops tasks so I haven't committed this back yet.

Reply:
On Cactus, see /lfs/h2/emc/global/noscrub/Kate.Friedman/expdir/devv16_192/config.vrfy (tracker section) for the consolidation I did.

Reply:
At time this PR/issue created, the information in question was not available. We can modify it when now solution is tested.